### PR TITLE
reword pipe header, change binding implementation

### LINKED ISSUE #341: /pipes-adding-a-pipe and /pipes-using-our-pipe: Title and code
- We should “Using a pipe”, not Adding
- `id=”{{i}}”` should be `[id]=”i”` (but `id` could probably be removed entirely)

### PR DESCRIPTION
**Pull request by jamesbs**
Closes #341

### CODE REVIEW

File: slides/src/content/pipes.md [hunk 1]
```diff
@@ -24,15 +24,15 @@
...
+## Using a Pipe
 
 - Put the name of the pipe inside `{{...}}`
 - Use vertical bar `|` as separator
 
 #####_src/app/to-do-list/to-do-list.component.html_
 ```html
 <ul>
-  <li *ngFor="let item of thingsToDo; let i = index" id="{{i}}">
+  <li *ngFor="let item of thingsToDo; let i = index" [id]="i">
```
Comment (gvwilson) [inline slides/src/content/pipes.md]: did you submit a PR for this to https://github.com/gvwilson/angular-todo-example ?

**Comment by gvwilson** [comment]
ping @jamesbs

File: slides/src/content/pipes.md (hunk 1 above)
Comment (gvwilson) [inline slides/src/content/pipes.md]: I have not, i'm guessing i should I update the example as well

Comment (jamesbs) [inline slides/src/content/pipes.md]: last one was me

**Pull request merged**